### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by jksarsonas**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by jksarsonas — COMMENTED** [review]
the diff between old and new code is great feature to know the changes

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (jksarsonas) [inline index.html:50]: added refresh to play again

Comment (jksarsonas) [inline index.html:50]: another test

File: index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
```
Comment (jaik3n2th) [inline index.html:50]: add comment to make it understandable

File: index.html (hunk 1 above)
Comment (jaik3n2th) [inline index.html:50]:
```suggestion
<h2 hidden>Game over, refresh to play again. enjoy 🧑‍💻 🤖!</h2>
```

**Pull request merged**